Upstream-Status: Pending

Without this it fails to link against libtermcap causing various missing
symbols issues.

RP - 8/10/08

Support 6.3 which uses configure.ac rather than configure.in.
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 configure.ac | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/configure.ac b/configure.ac
index cea8f91..9075b8f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -218,10 +218,10 @@ if test -f ${srcdir}/support/shobj-conf; then
         AC_MSG_CHECKING(configuration for building shared libraries)
         eval `TERMCAP_LIB=$TERMCAP_LIB ${CONFIG_SHELL-/bin/sh} ${srcdir}/support/shobj-conf -C "${CC}" -c ${host_cpu} -o ${host_os} -v ${host_vendor}`
 
-#	case "$SHLIB_LIBS" in
-#	*curses*|*termcap*|*termlib*)	;;
-#	*)			SHLIB_LIBS="$SHLIB_LIBS $TERMCAP_LIB" ;;
-#	esac
+	case "$SHLIB_LIBS" in
+	*curses*|*termcap*|*termlib*)	;;
+	*)			SHLIB_LIBS="$SHLIB_LIBS $TERMCAP_LIB" ;;
+	esac
 	
         AC_SUBST(SHOBJ_CC)
         AC_SUBST(SHOBJ_CFLAGS)
-- 
1.8.1.2

